scrooge-generator/src/main/resources/scalagen/finagleService.scala

```diff
@@ -119,6 +120,7 @@ class {{ServiceName}}$FinagleService(
   // ---- end boilerplate.
 
 {{/hasParent}}
+  private[this] val scopedStats = if (serviceName != "") stats.scope(serviceName) else stats
```

Comment:
currently, all the server related stats are scoped with name. following are examples from my server, so I think we just follow the convention
"srv/XXServer/closechans": 84,
"srv/XXServer/closed": 84,
"srv/XXServer/closes": 0,